pilot/pkg/networking/plugin/authz/rbac.go

```diff
@@ -468,6 +481,128 @@ func buildHTTPFilter(service *serviceMetadata, option rbacOption, is11 bool) *ht
 	return out
 }
 
+// convertRbacRulesToFilterConfigV2 is the successor of convertRbacRulesToFilterConfig, which supports
+// converting AuthorizationPolicy.
+func convertRbacRulesToFilterConfigV2(service *serviceMetadata, option rbacOption) *http_config.RBAC {
```

Comment:
Please don't do this, instead, create a new function in a new file just for the v2. It's okay to duplicate some code if necessary. When v2 is out, v1 will be in maintain mode and only bug fix will go to v1.

Reply:
Moved. Please take a look :) Thanks Yangmin